winterfell/webapp/src/js/noangular/signin.js

```diff
@@ -49,15 +74,15 @@ $(document).ready(function(){
     $('.inputs').removeClass('vfi-show');
   });
 
-  $('#signup-view .submit-btn').click(function(e) {
+  $('#signup-view .inputs form').submit(function(e) {
```

Comment:
I'm actually going to keep it with jquery.
There are a lot of things that are just easier with jquery like a POST request, grabbing values from elements, setting text for certain elements, adding/removing classes, etc.
Since most sites use jQuery anyway, I don't think it's super critical to optimize to the point of using vanilla over jQuery.